Backport jetpack 15.2-beta Changes (#45745)
Automattic/jetpack: projects/packages/forms/CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [6.14.0] - 2025-11-03
### Added
- Device detection: Add a method that returns the browser display name. [#45674]
- Add "auto" width for fields and detect when a form has a single input in it to show prettier validation errors. [#45689]

> - Add a new jwt filter. [#45731]

Reviewer: ```suggestion
- Add a new JWT filter. [#45731]
```

- Add emoji flag to the response email next to the IP Address. [#45645]
- Add error codes to submission errors. [#45628]
- Add first and last name variations. [#45517]
- Add flag next to the IP address. [#45643]
- Add time field. [#45665]
- Add visibility support for field labels. [#45722]

> - Enable forms push notifications functionality.. [#45682]

Reviewer: ```suggestion
- Enable forms push notifications functionality. [#45682]
```

- Link IP to lookup service. [#45653]
- Release Image Select Field to production. [#45577]

> - Start storing the country code with the form responses based on the IP address. [#45599]

Reviewer: ```suggestion
- Store the country code with the form responses based on the IP address. [#45599]
```

### Changed
- Add browser info to form responses exports. [#45709]
- Add details for JWT decoding errors. [#45646]
- Add the IP address as a default field in the forms dashboard. [#45704]
- Create new integrations list component. [#45648]
- Default to notify the post author when the notification recipients list is empty. [#45633]
- Disable Jetpack CRM integration by default. [#45667]
- Move storage setting lower in sidebar. [#45727]

> - Revert CRM frontend default to true. [#45669]

Reviewer: This change was reverted, so let's remove it:
```suggestion
- Move storage setting lower in sidebar. [#45727]
```

- Show gravatars also for rows without emails. [#45726]
- Use a dropdown selector to list eligible users to receive form notifications. [#45634]

> - Use jetpack logo on dashboard header. [#45609]

Reviewer: ```suggestion
- Use Jetpack logo on dashboard header. [#45609]
```

- Use table to render response meta. [#45656]
- Improve empty state centering and fix scrolling by consolidating layout min-height rules and improving flexbox handling. [#45649]
- Update package dependencies. [#45652]

### Fixed
- Fix a bug with the default email in the editor. [#45711]
- Fix a consoler error triggered by iterable without key prop. [#45690]
- Fix an error on phone field controls due to lack of key prop. [#45659]
- Fix react error due to nesting tr as direct descendant of table, needs tbody. [#45691]
- Fix response selection filtering bug caused by type mismatch. [#45611]
- Fix tooltip in response IP link. [#45705]

> - Fix where notices show up so they don't overlap the bottom action bar. [#45616]

Reviewer: ```suggestion
- Fix a console error triggered by iterable without key prop. [#45690]
- Fix an error on phone field controls due to lack of key prop. [#45659]
- Fix React error due to missing tbody when nesting tr as direct descendant of table. [#45691]
- Fix response selection filtering bug caused by type mismatch. [#45611]
- Fix tooltip in response IP link. [#45705]
- Ensure notices don't overlap the bottom action bar. [#45616]
```
